modin/core/execution/unidist/implementations/pandas_on_unidist/partitioning/virtual_partition.py

```diff
@@ -62,6 +52,33 @@ class PandasOnUnidistDataframeVirtualPartition(PandasDataframeAxisPartition):
     instance_type = unidist.core.base.object_ref.ObjectRef
     axis = None
 
+    # these variables are specially initialized at runtime (#6023)
```

Comment:
```suggestion
    # these variables are intentionally initialized at runtime (see #6023)
```

btw, did we have an issue in master for unidist?

Reply:
Actually no, I just did the same as for Ray.